sdk/search/Azure.ResourceManager.Search/CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
 
 ### Other Changes
 
+## 1.0.2 (2023-03-10)
```

Comment:
This is in wrong structure. We should not assume the release date here. Please remove this line.
The changelog should always have this structure:
```
## {version number} ({release date})
### Features Added
### Breaking Changes
### Bugs Fixed
### Other Changes
```
With this line, this version no longer has the four entries.

Reply:
And, in addition, because new APIs are introduced in your version and NO breaking changes are introduced, we MUST bump the minor version instead of the patch version.
The new version should be `1.1.0`. Please make sure you change the version number in csproj file as well